src/data/markdown/docs/30 xk6-browser/02 API/03 BrowserType/launch--options--.md

```diff
@@ -13,12 +13,62 @@ Launches a new browser process.
 | env               | string[] | `null`  | Environment variables to set before launching browser process.                                                                                                                                                                                                        |
 | executablePath    | string   | `null`  | Override search for browser executable in favor of specified absolute path.                                                                                                                                                                                           |
 | headless          | boolean  | `true`  | Show browser GUI or not.                                                                                                                                                                                                                                              |
-| ignoreDefaultArgs | string[] | `null`  | Ignore any of the default arguments included when launching browser process.                                                                                                                                                                                          |
+| ignoreDefaultArgs | string[] | `null`  | Ignore any of the [default arguments](#default-arguments) included when launching browser process.                                                                                                                                                                                          |
 | proxy             | string   | `null`  | Specify to set browser's proxy configuration.                                                                                                                                                                                                                         |
 | slowMo            | string   | `null`  | Slow down input actions and navigation by the specified time e.g. `'500ms'`.                                                                                                                                                                                          |
 | timeout           | string   | `'30s'` | Default timeout to use for various actions and navigation.                                                                                                                                                                                                            |
 
 
+### Default arguments
```

Comment:
This way, it can be added to the side table of contents. Currently, we have Examples only.
<img width="176" alt="Screenshot 2023-02-10 at 14 43 38" src="https://user-images.githubusercontent.com/10811379/218119812-16316f38-216c-4304-bd0a-ef4162493f09.png">